core/helpers/EEH_Event_Query.helper.php

```diff
@@ -541,9 +541,12 @@ public static function posts_where_sql_for_show_expired($show_expired = false)
     public static function posts_where_sql_for_event_category_slug($event_category_slug = null)
     {
         global $wpdb;
-        return ! empty($event_category_slug)
-            ? $wpdb->prepare(" AND {$wpdb->terms}.slug = %s ", $event_category_slug)
-            : '';
+        if (! empty($event_category_slug) ) {
+            $event_category_slugs_array = array_map('trim', explode(',', $event_category_slug));
+            $event_category_slugs_prepare = implode( ', ', array_fill( 0, count( $event_category_slugs_array ), '%s' ));
+            return $wpdb->prepare(" AND {$wpdb->terms}.slug IN ($event_category_slugs_prepare) ", $event_category_slugs_array);
```

Comment:
`$event_category_slugs_array` needs to be imploded here as well no?

Reply:
Yeah, good shout! Done.